app/services/create_question.rb

```diff
@@ -6,19 +6,21 @@ class CreateQuestion
   include Callable
 
   attr_reader :question, :type, :tags, :question_options
-  def initialize(question_params)
-    @question = question_params[:question]
-    @type = question_params[:type]
-    @tags = question_params[:tags]
-    @question_options = question_params[:question_options]
+  def initialize(params)
+
+    @question_node = JSON.parse(params[:question_node], symbolize_names: true)
```

Comment:
I'm thinking about having a service parsing out SlateJs objects because we will have to do the same `JSON.parse` for `question_options_arr`.